draft-thomson-webpush-protocol.xml

```diff
@@ -328,7 +338,40 @@ Cache-Control: max-age:864000, private
 ]]></artwork>
       </figure>
     </section>
+    <section anchor="receipt_subscription" title="Subscribing to Receipts">
+      <t>
+        An application server sends a POST request to its configured push
```

Comment:
We can't have the application server configured in this fashion.  It needs to learn this URL from the user agent.